.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+---
+# Run the checkrefs.py script and check for typos.
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  workflow_dispatch:
+jobs:
+  python-script:
+    name: 'Checkrefs.py: Actors, Templates and Maps'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get latest release tag of 0 A.D.
+        id: tag_name
+        run: |
+          echo "release_tag=$(curl -s https://api.github.com/repos/0ad/0ad/releases/latest | jq -r '.tag_name')" >> $GITHUB_OUTPUT
+      - name: Check out 0 A.D. (${{ steps.tag_name.outputs.release_tag }})
+        uses: actions/checkout@v3
+        with:
+          repository: 0ad/0ad
+          path: 0ad_directory
+          ref: ${{ steps.tag_name.outputs.release_tag }}
+          clean: true
+      - name: Check out community maps 2
+        uses: actions/checkout@v3
+        with:
+          path: community_maps_2_directory
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.x
+      - run: pip install lxml
+      - name: Run Python script
+        run: |-
+          cd "$GITHUB_WORKSPACE/0ad_directory/source/tools/entity" || exit 1
+          array=()
+          while IFS='' read -r line; do array+=("$line"); done < <(python checkrefs.py -m "$GITHUB_WORKSPACE/community_maps_2_directory" -tax 2>&1)
+          for i in "${array[@]}"; do
+            [[ $i =~ ^WARNING ]] && echo "::warning:: $i"
+            [[ $i =~ ^ERROR ]] && echo "::error:: $i"
+          done
+          [[ "${array[*]}" =~ (WARNING|ERROR) ]] && exit 1 || exit 0
+  typos:
```

Comment:
Let's not include this. At this time,  I don't think this action is very necessary for this type of repository.